jdk/test/sun/security/pkcs11/SecmodTest.java

```diff
@@ -90,6 +96,25 @@ private static void copyFile(String name, String srcDir, String dstDir) throws I
         out.close();
     }
 
+    private static void deleteDir(final Path directory) throws IOException {
```

Comment:
Can we not use `FileUtils.deleteFileTreeWithRetry(Path)` here? That seems to be the same thing, but with better error handling.

See `jdk/test/lib/jdk/test/lib/util/FileUtils.java`